src/local-ssh/proxy.ts

```diff
@@ -249,6 +252,9 @@ class WebSocketSSHProxy {
     }
 
     async sendUserStatusFlow(status: 'connected' | 'connecting' | 'failed') {
+        this.metricsReporter.reportConnectionStatus(this.options.host, status, this.flow.failureCode).catch(_e => {
```

Comment:
`this.flow.gitpodHost`?

Reply:
They are the same

Reply:
Alright, good if we keep use gitpodHost from flow everywhere

Reply:
true, within the proxy it cannot vary within the same process?

Reply:
> within the proxy it cannot vary within the same process?

Yes, `gitpodHost` in proxy.ts is passed by ssh proxy command itself which is constant in one proxy process